packages/core/core/src/CommittedAsset.js

```diff
@@ -119,6 +122,25 @@ export default class CommittedAsset {
     return this.map;
   }
 
+  getSourcesContent(): Promise<?SourcesContentDictionary> {
+    if (this.sourcesContent == null) {
+      if (this.value.sourcesContentKey != null) {
+        this.sourcesContent = (async () => {
```

Comment:
I think this will result in uncaught promise exceptions. Any reason why getSourcesContent isn't async?

Reply:
Based it on getMap which has this same pattern, I thought it was so if there's a promise already it can return the promise instead of refetching the data. However I don't really think this would ever really occur in reality